tox.ini

```diff
@@ -8,7 +8,7 @@ envlist = py{27}-django{18,111}
 deps =
     -rrequirements.txt
     -rtest_requirements.txt
-    -egit+https://github.com/edx/xblock-sdk.git#egg=xblock-sdk
+    -egit+https://github.com/edx/xblock-sdk.git@48cb0d5066a1f5ab8eecba41bb3d63c78c985a81#egg=xblock-sdk
```

Comment:
Something about https://github.com/edx/xblock-utils/pull/45 caused tox to return this error, so I've reverted to the revision just prior.
```
django18 runtests: commands[3] | python ./run_tests.py --with-coverage --cover-package=xblockutils
usage: run_tests.py test [-h] [--version] [-v {0,1,2,3}] [--settings SETTINGS]
                         [--pythonpath PYTHONPATH] [--traceback] [--no-color]
                         [--noinput] [--failfast] [--testrunner TESTRUNNER]
                         [--liveserver LIVESERVER] [-t TOP_LEVEL] [-p PATTERN]
                         [-k] [-r] [-d]
                         [test_label [test_label ...]]
run_tests.py test: error: unrecognized arguments: --with-coverage --cover-package=xblockutils
ERROR: InvocationError: '/home/travis/build/edx/xblock-utils/.tox/django18/bin/python ./run_tests.py --with-coverage --cover-package=xblockutils'
```

Some version incompatibility introduced with `nose`, `coverage`, or other?